notebooks/segmentation/segmentation_inspection.ipynb

```diff
@@ -231,9 +320,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%time\n",
-    "by_hand_thumbnail = video_generator.get_thumbnail_video(origin=(100, 200), frame_shape=(64, 64),\n",
-    "                                                        quality=5)"
+    "roi_8 = roi_lookup[8]"
```

Comment:
May want to consider renaming this variable?  The absolute first thing that I did was try an ROI other than '8'.